.coveragerc

```diff
@@ -93,9 +93,7 @@ omit =
     homeassistant/components/*/pilight.py
 
     homeassistant/components/knx.py
-    homeassistant/components/switch/knx.py
-    homeassistant/components/binary_sensor/knx.py
-    homeassistant/components/thermostat/knx.py
+    homeassistant/components/*/knx.py
```

Comment:
Without a real KNX gateway or a simulated gateway it will be quite complicated to implement units tests. 
Implementing a gateway simulation just for unit tests is really complicated as the KNX standard isn't trivial.